packages_rs/nextclade-web/src/components/Common/BrowserWarning.tsx

```diff
@@ -21,20 +26,37 @@ export function BrowserWarning() {
     }
 
     const { name, version } = browser.getBrowser()
-    const nameAndVersion = [name, version].filter(notUndefinedOrNull).join(' ')
-
-    return t(
-      `This browser version (${nameAndVersion}) is not supported. Nextclade works best in the latest version of Chrome or Firefox.`,
-    )
-  }, [t])
+    return [name, version].filter(notUndefinedOrNull).join(' ')
+  }, [])
 
-  if (!warningText) {
+  if (!nameAndVersion) {
     return null
   }
 
   return (
-    <UncontrolledAlert color="warning" className="text-center m-0">
-      {warningText}
-    </UncontrolledAlert>
+    <Modal centered isOpen={isOpen} backdrop="static" toggle={dismiss} fade={false} size="lg">
+      <ModalHeader toggle={dismiss} tag="div">
+        <h4 className="text-center text-danger">{t('Unsupported browser')}</h4>
+      </ModalHeader>
+
+      <ModalBody>
+        <p>{t('This browser version ({{nameAndVersion}}) is not supported.', { nameAndVersion })}</p>
+        <p>
+          {t('Nextclade works best in the latest version of ')}
+          <LinkExternal href="https://www.google.com/chrome/">{t('Chrome')}</LinkExternal>
+          <span>{t(' or ')}</span>
+          <LinkExternal href="https://www.mozilla.org/firefox/browsers/">{t('Firefox')}</LinkExternal>
+          <span>{t('.')}</span>
+        </p>
```

Comment:
```suggestion
        </p>
        <p>
          {t('In addition to Chrome and Firefox, Nextclade should work with any Chromium based browser, such as Edge, Brave, Opera etc.')}
        </p>
        <p>
          {t('If you are shown this modal even though you're using a Chromium-based or Firefox browser, please let us know by opening an issue ')}
          <LinkExternal href="https://github.com/nextstrain/nextclade/issues/new?assignees=&labels=t%3Abug%2C+help+wanted%2C+good+first+issue%2C+needs+triage&template=01_report_bug.md">{t('here')}</LinkExternal>
          <span>{t('.')}</span>
        </p>
```

Reply:
Regarding browsers, I think this is unnecessary. We are not here to teach people browsers landscape or promote anything. We just need the app to be usable.

Those who know, they don't need to be told. Those who don't know will likely only be confused by these extra details.

And again, this will not fit well on mobile.